modules/database/src/ioc/db/dbChannel.h

```diff
@@ -62,7 +64,8 @@ typedef struct dbChannel {
     ELLLIST post_chain;       /* list of filters to be called post-event-queue */
 } dbChannel;
 
-/* Prototype for the channel event function that is called in filter stacks
+/**
+ * Prototype for the channel event function that is called in filter stacks
```

Comment:
For the brief description, I think with the way the Doxyfile is configured, you have to prepend it with the `\brief` command. Else the brief description will not show up in indexes and such.